code/datums/votes/map_vote.dm

```diff
@@ -63,6 +66,23 @@
 	message = initial(message)
 	return TRUE
 
+/// Before we create a vote, remove all maps from our choices that are outside of our population range. Note that this can result in zero remaining choices for our vote, which is not ideal. Let's warn users about that if it comes up.
```

Comment:
we don't warn users in this proc, why mention it in the autodoc? surely this is better done elsewhere